aws/resource_aws_lb_test.go

```diff
@@ -1770,6 +1804,46 @@ resource "aws_eip" "lb" {
 `, lbName)
 }
 
+func testAccAWSLBConfig_networkLoadBalancerPrivateIPV4Address(lbName string) string {
+	return fmt.Sprintf(`
+resource "aws_lb" "lb_test" {
+  name               = "%s"
+  internal           = true
+  load_balancer_type = "network"
+
+  enable_deletion_protection       = false
+
+  subnet_mapping {
+	subnet_id            = "${aws_subnet.alb_test.id}"
+	private_ipv4_address = "10.10.0.15"
+  }
+
+  tags = {
+    Name = "TestAccAWSALB_privateipv4address"
+  }
+}
+
+resource "aws_vpc" "alb_test" {
+  cidr_block = "10.10.0.0/16"
+
+  tags = {
+    Name = "TestAccAWSALB_privateipv4address"
+  }
+}
+
+resource "aws_subnet" "alb_test" {
+  vpc_id                  = "${aws_vpc.alb_test.id}"
+  cidr_block              = "10.10.0.0/21"
+  map_public_ip_on_launch = true
+  availability_zone       = "us-west-2a"
```

Comment:
to make test region agnostic and to avoid environment specific failures, can please add the following to the test config:

```
data "aws_availability_zones" "available" {
  state = "available"

  filter {
    name   = "opt-in-status"
    values = ["opt-in-not-required"]
  }
}
```

and change the hardcoded az to `"${data.aws_availability_zones.available.names[0]}"`